lib/msal-common/src/authority/Authority.ts

```diff
@@ -1314,4 +1314,20 @@ export class Authority {
 
         return ciamAuthority;
     }
+
+    /**
+     * Extract tenantId from authority
+     */
+    static getTenantFromAuthorityString(authority: string): string | undefined {
```

Comment:
Instead of static function can we pull this out of the class? Functions > static functions for minification. You can export it at the bottom of this file or create a new file for "AuthorityUtils" (or pick another name if you're tired of "Utils" ;) )